### ISSUE TITLE
deprecated IPMPControlBox

### ISSUE THREAD

**Issue by podborski** [MPEGGroup/FileFormat#79]
ISOBMFF has the following text:

> When IPMP is used outside of MPEG-4 systems, then a ‘global’ IPMPControlBox may also occur within the MovieBox

As well as:

> `IPMPControlBox		IPMP_control;			// optional`

while `IPMPControlBox` is clause 8.12.5 was deprecated. Should it be cleaned up?

**Comment by jeanlf** [comment]
yes, as well as mentions to IPMP boxes in 8.12.1 - we should only have pure IPMP (MPEG-4 systems) without IPMP boxes

**Comment by cconcolato** [comment]
The group agrees to move every mention of IPMP to MPEG-4 Part 14.

**Comment by podborski** [comment]
I guess this relates to this https://github.com/MPEGGroup/FileFormat/issues/110

**Comment by cconcolato** [comment]
The group currently does not have an on-going amendment to Part 14 and no place to put this text. However, a new amendment may start soon (e.g. to fix some 'template' or SDL behavior) and in that case we should move the IPMP text out of Part 12 into Part 14.
